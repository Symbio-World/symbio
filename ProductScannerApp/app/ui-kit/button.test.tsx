import 'react-native'
import React from 'react'
import { render, fireEvent } from 'react-native-testing-library'
import { Button } from './button'

describe('Error', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<Button />)
    expect(toJSON()).toMatchSnapshot()
  })
})
