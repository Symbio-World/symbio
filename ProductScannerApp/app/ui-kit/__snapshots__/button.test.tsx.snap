// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#48bb78",
      "borderRadius": 4,
      "display": "flex",
      "justifyContent": "center",
      "opacity": 1,
      "padding": 16,
      "textAlignVertical": "center",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "textAlign": "center",
        },
        Object {
          "fontSize": 24,
        },
        Object {
          "color": "#fff",
        },
      ]
    }
  />
</View>
`;
